src/main/java/Model/Gear.java

```diff
@@ -0,0 +1,21 @@
+package Model;
+
+import javax.persistence.*;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+@Entity
+public class Gear {
+
+    @Id
+    private Integer id;
+    private Enum<GearType> type;
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "RentSpot_id")
+    private RentSpot rentSpot = new RentSpot();
```

Comment:
Dlaczego jest tutaj tworzony nowy `RentSpot`?

Reply:
na tym stanęliśmy przy zakończeniu dnia. Ostatni commit aktualny do merge'a

Reply:
Tutaj raczej nie chcemy tworzyć spota, więc wywaliłbym to `= new RentSpot()` a poza tym cała reszta wygląda ok :)